packages/flutter/test/material/dropdown_theme_test.dart

```diff
@@ -0,0 +1,633 @@
+        autofocus: autofocus,
+        focusColor: focusColor,
+        dropdownColor: dropdownColor,
+        items: listItems,
+        selectedItemBuilder: selectedItemBuilder,
+        itemHeight: itemHeight,
+        alignment: alignment,
+        menuMaxHeight: menuMaxHeight,
+      ),
+    );
+  }
+  return DropdownButton<String>(
+    key: buttonKey,
+    value: value,
+    hint: hint,
+    disabledHint: disabledHint,
+    onChanged: onChanged,
+    onTap: onTap,
+    icon: icon,
+    iconSize: iconSize,
+    iconDisabledColor: iconDisabledColor,
+    iconEnabledColor: iconEnabledColor,
+    isDense: isDense,
+    isExpanded: isExpanded,
+    underline: underline,
+    focusNode: focusNode,
+    autofocus: autofocus,
+    focusColor: focusColor,
+    dropdownColor: dropdownColor,
+    items: listItems,
+    selectedItemBuilder: selectedItemBuilder,
+    itemHeight: itemHeight,
+    alignment: alignment,
+    menuMaxHeight: menuMaxHeight,
+  );
+}
+
+Widget buildFrame({
+  Key? buttonKey,
+  String? value = 'two',
+  ValueChanged<String?>? onChanged,
+  VoidCallback? onTap,
+  Widget? icon,
+  Color? iconDisabledColor,
+  Color? iconEnabledColor,
+  double iconSize = 24.0,
+  bool isDense = false,
+  bool isExpanded = false,
+  Widget? hint,
+  Widget? disabledHint,
+  Widget? underline,
+  List<String>? items = menuItems,
+  List<Widget> Function(BuildContext)? selectedItemBuilder,
+  double? itemHeight = kMinInteractiveDimension,
+  AlignmentDirectional alignment = AlignmentDirectional.centerStart,
+  TextDirection textDirection = TextDirection.ltr,
+  Size? mediaSize,
+  FocusNode? focusNode,
+  bool autofocus = false,
+  Color? focusColor,
+  Color? dropdownColor,
+  bool isFormField = false,
+  double? menuMaxHeight,
+  Alignment dropdownAlignment = Alignment.center,
+}) {
+  return TestApp(
+    textDirection: textDirection,
+    mediaSize: mediaSize,
+    child: Material(
+      child: Align(
+        alignment: dropdownAlignment,
+        child: RepaintBoundary(
+          child: buildDropdown(
+            isFormField: isFormField,
+            buttonKey: buttonKey,
+            value: value,
+            hint: hint,
+            disabledHint: disabledHint,
+            onChanged: onChanged,
+            onTap: onTap,
+            icon: icon,
+            iconSize: iconSize,
+            iconDisabledColor: iconDisabledColor,
+            iconEnabledColor: iconEnabledColor,
+            isDense: isDense,
+            isExpanded: isExpanded,
+            underline: underline,
+            focusNode: focusNode,
+            autofocus: autofocus,
+            focusColor: focusColor,
+            dropdownColor: dropdownColor,
+            items: items,
+            selectedItemBuilder: selectedItemBuilder,
+            itemHeight: itemHeight,
+            alignment: alignment,
+            menuMaxHeight: menuMaxHeight,
+          ),
+        ),
+      ),
+    ),
+  );
+}
+
+class TestApp extends StatefulWidget {
```

Comment:
This class as well does not appear to be used, except by the buildFrame method that doesn't appear to be in use.

Reply:
Good catch